server/src/main/java/org/elasticsearch/common/file/AbstractFileWatchingService.java

```diff
@@ -195,7 +197,7 @@ private void startIfMaster(ClusterState clusterState) {
         }
     }
 
-    final synchronized void startWatcher(ClusterState clusterState) {
+    public final synchronized void startWatcher(ClusterState clusterState) {
```

Comment:
This is just for tests right? I don't think it should be public. Can the tests use the higher level clusterChanged method to trigger this indirectly?